test/libyul/yulOptimizerTests/fullSuite/unusedFunctionParameterPruner.yul

```diff
@@ -27,7 +27,7 @@
 //     {
 //         out1 := sload(mload(32))
 //         out2 := add(out1, 1)
-//         extcodecopy(out1, out2, 1, b)
+//         mcopy(out2, out1, b)
```

Comment:
Oh, CI is failing because this is only available on Cancun. I guess `calldatacopy(out2, out1, b)` should work too.